docs/source/contrib.mue.rst

```diff
@@ -0,0 +1,44 @@
+MuE
+===
```

Comment:
Could we make this more descriptive, say "Sequence alignment with MuE"?  This title appears in the docs.pyro.ai sidebar, so you'll it would be good to advertise it to people who don't already know what MuE means 😉